### ISSUE TITLE
Suggestions for 2017

### ISSUE THREAD

**Issue by hackygolucky** [cascadiajs/2016.cascadiajs.com#46]
### To the future!

We heard a ton of great ideas this week about future community iterations. Let's capture it here!

**Comment by blakmatrix** [comment]
An observation: The campfires were a big hit, it may be useful in the future to create the environment for some small impromptu intimate gatherings like there can be had around a fire--in the case of a burn ban (or regardless) we could use the volunteers and/or select attendees (or just a couple of spaces) to initiate small group communication between people on the first evening in a relatively quiet setting set up for that.

**Comment by coldpour** [comment]
@blakmatrix agree. it was awesome that the canadians showed up with extra instruments, but I bet if we could provide some instruments around the fire, we wouldn't need to hope they showing up to play music.

**Comment by KatieK2** [comment]
Was thinking that brownies and glowstick orbs could be a good substitute for fire and s'mores. People always come to light and snacks.

**Comment by KatieK2** [comment]
- Have an "Apprentice.js" event for folks in their first year of programming, and those who want to help them out. (Pretty sure this was both Dave and someone else's idea.)

**Comment by Qard** [comment]
With all those extra open rooms available it might be interesting the add an unconf-style discussion group track. We could have a rotating 3-way thing going with the 3 topics (css, browser, node) and the 3 session types (talk, workshop, discussion group) so topics don't overlap. Each day, between the 3 session types, they'd cover all 3 of the topics.

**Comment by jasonrhodes** [comment]
Consider having the event 4-10 times a year instead of once.

**Comment by KatieK2** [comment]
@jasonrhodes - That's called SeattleJS. It happened last night.

**Comment by revin** [comment]
I _really_ wish there was a way to have it out in the San Juans somewhere (but there aren't any big enough venues), or even Victoria. Probably not realistic, but jotting it down anyway in case someone who can make that happen happens to see it.

**Comment by jeanne-m** [comment]
Have Marian officially run the karaoke with her selection.

**Comment by stebogit** [comment]
any updates?
